graylog2-web-interface/src/pages/SearchPage.jsx

```diff
@@ -30,6 +30,7 @@ const SearchPage = React.createClass({
     Reflux.connect(CurrentUserStore),
     Reflux.listenTo(InputsStore, '_formatInputs'),
     Reflux.listenTo(RefreshStore, '_setupTimer', '_setupTimer'),
+    Reflux.listenTo(DecoratorsStore, '_refreshDataFromDecoratorStore', '_refreshDataFromDecoratorStore')
```

Comment:
Strange, the other examples on that page also use normal function references instead of strings.

But for the sake of conformity with the rest of our codebase, using single-quoted strings is fine.